backend/tracim_backend/config.py

```diff
@@ -483,16 +483,16 @@ def __init__(self, settings: typing.Dict[str, typing.Any]):
             'caldav.radicale_proxy.base_url',
             None
         )
-        self.CALDAV_RADICALE_CALENDAR_DIR = 'calendar'
+        self.CALDAV_RADICALE_AGENDA_DIR = 'agenda'
```

Comment:
is this `calendar` folder name is standard for radicale, it may stay `calendar` (if no specific value, keep agenda)

Reply:
it is not that just a config to set first level dir for both proxy and radicale :
`/agenda` in tracim should match `/agenda` in radicale.